cypress/support/commands.js

```diff
@@ -177,3 +177,47 @@ Cypress.Commands.add('waitUntilMail', (alias) => {
       .then((mails) => mails?.length > 0),
   );
 });
+
+Cypress.Commands.add('createEvent', (data) => {
+  const eventMutation = {
+    operationName: 'createEvent',
+    variables: {
+      data: { ...data },
+    },
+    query: `mutation createEvent($data: CreateEventInputs!) {
+      createEvent(data: $data) {
+        id
+      }
+    }`,
+  };
+  cy.login();
```

Comment:
```suggestion
```
Since, in part, we're trying to improve performance, we shouldn't login here.  It's a little less DRY, but means that the command is clearer (`createEvent()` should just create an event).

Reply:
I'm not a fan of this either. Problem is request needs to have token added, without calling login here, it needs to be ensured token is there. Maybe then pass the authorization token when calling function? `createEvent(data, token)`
```

Reply:
Right, we definitely need to be logged in when calling this. That said, I think we can separate the calls and do this:

```js
cy.login();
...
cy.createEvent(eventData);
```

In this spec file we're already logging in the `beforeEach` hook, so we're good.

Also, if we _do_ forget, we get a nice error in the console: `Error: User must be logged in to create events`